### DISCUSSION TITLE
In your mobile browser, clicking on the wallet automatically redirects you to the dapp browser within the wallet

### DISCUSSION THREAD

**Discussion by lililemon** [rainbow-me/rainbowkit#1949] [Ideas]
Each wallet can be freely configured to jump directly to the dapp browser within the wallet by clicking on the wallet when accessed from a mobile browser

**Comment by lililemon** [comment]
This may drastically improve the user experience without requiring the user to repeatedly switch between the two apps

**Comment by 0xmkh** [comment]
@lililemon I think you're talking about the coinbase wallet right ? Some wallets have their own behaviours which is out of our control, but if that's happening with coinbase wallet we can find a solution for that.

**Reply by lililemon** [reply]
No, I've found that WalletButton.Custom can do this, it just needs to determine if it's within the mobile wallet browser when clicked on
